src/main/java/baritone/pathing/movement/movements/MovementAscend.java

```diff
@@ -68,7 +68,7 @@ protected Set<BetterBlockPos> calculateValidPositions() {
     public static double cost(CalculationContext context, int x, int y, int z, int destX, int destZ) {
         IBlockState toPlace = context.get(destX, y, destZ);
         double additionalPlacementCost = 0;
-        if (!MovementHelper.canWalkOn(context.bsi, destX, y, destZ, toPlace)) {
+        if (!context.precomputedData.canWalkOn(context.bsi, x, y, z, toPlace)) {
```

Comment:
Changing this from `destX` to `x` doesn't look right

Reply:
It's not right, thanks for pointing this out.